packages/QueryEngine/tests/Unit/Core/Engine/GuardEvaluatorTest.php

```
@@ -0,0 +1,271 @@
+        $context = $this->createContext();
+
+        $this->expectException(GuardConditionFailedException::class);
+
+        $evaluator->evaluateAll([
+            'time_window' => [
+                'type' => 'time_window',
+                'start' => (new \DateTimeImmutable('+10 minutes'))->format(\DateTimeInterface::ATOM),
+                'end' => (new \DateTimeImmutable('-10 minutes'))->format(\DateTimeInterface::ATOM),
+            ],
+        ], $context);
+    }
```

PHPMD (2.15.0)

[error] The method it_rejects_time_window_guard_when_start_is_after_end is not named in camelCase. (undefined)

(CamelCaseMethodName)